src/rotation.zig

```diff
@@ -126,7 +126,7 @@ pub const RotationMode = struct { // MARK: RotationMode
 	// Rotates block data counterclockwise around the Z axis.
 	rotateZ: *const fn(data: u16, angle: Degrees) u16 = DefaultFunctions.rotateZ,
 
-	createBlockModel: *const fn(zon: ZonElement) ModelIndex = &DefaultFunctions.createBlockModel,
+	createBlockModel: *const fn(blockId: []const u8, zon: ZonElement) ModelIndex = &DefaultFunctions.createBlockModel,
```

Comment:
Why not pass the block directly, since that appears to be what you want to use here.